<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeveloperEditor</class>
 <widget class="QDialog" name="DeveloperEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Данные разработчика</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Имя:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameEdit">
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ставка:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="wageRateEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="gridVisible">
        <bool>true</bool>
       </property>
       <property name="verticalHeaderFormat">
        <enum>QCalendarWidget::NoVerticalHeader</enum>
       </property>
       <property name="navigationBarVisible">
        <bool>true</bool>
       </property>
       <property name="dateEditEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="exceptDayBox">
         <property name="title">
          <string>Исключение</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QDateEdit" name="dayDateCtrl">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="isWorkingDayCheck">
            <property name="text">
             <string>Рабочий день</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="exceptDayNameLbl">
              <property name="text">
               <string>Описание:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="exceptDayName"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="exceptTypeComboLbl">
              <property name="text">
               <string>Тип исключения:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="exceptTypeCombo"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setPeriodBtn">
         <property name="text">
          <string>Задать период...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeveloperEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
